docs/deployment-process/operations-runbooks/index.md

```diff
@@ -4,7 +4,7 @@ description: Runbooks can be used to automate routine or emergency operations-ce
 position: 140
 ---
 
-We're planning to introduce Runbooks in **Octopus 2019.10**.
+Runbooks were introduced in **Octopus 2019.10**.
```

Comment:
2019.11?

Reply:
Runbooks were introduced in 2019.10 behind this feature flag. 2019.11 is just where we are lifting the feature flag and tightening a few things.

Reply:
But if you want docs to help drive a version a customer will pick to use a feature, should it not be when the feature ships completely, not in EAP?

Reply:
I'll update this sentence to reflect when we lifted the feature flag (2019.11) so it's clear

Reply:
👍